pennylane/transforms/batch_transform.py

```diff
@@ -424,3 +424,68 @@ def _wrapper(dev):
 
     def _tape_wrapper(self, *targs, **tkwargs):
         return lambda tape: self.construct(tape, *targs, **tkwargs)
+
+
+def map_batch_transform(transform, tapes):
+    """Map a batch transform over multiple tapes.
+
+    Args:
+        transform (.batch_transform): the batch transform
+            to be mapped
+        tapes (Sequence[tapes]): The sequence of tapes the batch

```

Comment:
```suggestion
        tapes (Sequence[QuantumTape]): The sequence of tapes the batch
```
Since this way it'll link in the docs (I think?)

Reply:
thanks! good catch